public/app/features/provisioning/Wizard/FinishStep.tsx

```diff
@@ -52,15 +52,13 @@ export function FinishStep() {
             Changes in git will eventually be pulled depending on the synchronization interval.
           </Alert>
         )}
-        {!isPublic && (
```

Comment:
required for local config, and `isPublic` is really just a guess so it seems better to show this because this will add a task that is pulling every 60